### PR TITLE
Refactor code and file names

### PR DESCRIPTION
**Pull request by tyouwei**
Refactored the following details:
    1. All "address" has been changed to "intern" or "status"
    2. Changed some "person" into "application

### CODE REVIEW

**Review by AprupKale — COMMENTED** [review]
LGTM 👍! Ready to merge. Great work!

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ plugins {
     id 'jacoco'
 }
 
-mainClassName = 'seedu.address.Main'
+mainClassName = 'seedu.intern.Main'
```
Comment (tim-pipi) [inline build.gradle:9]: I think these may change in the future, `intern` is not confirmed yet, so maybe no need to change now? I saw this in a few other places too!

**Review by tim-pipi — COMMENTED** [review]
LGTM! Just concerns about the name changes from `address` to `intern`

File: build.gradle (hunk 1 above)
Comment (tyouwei) [reply build.gradle:9]: think we can leave it as is first, when the time comes I can change it to whatever we want

**Pull request merged**